app/controllers/chapters_controller.rb

```diff
@@ -286,4 +286,11 @@ def chapter_params
                                     author_attributes: [:byline, ids: [], coauthors: []])
 
   end
+
+  def speculative_loading?
+    sec_purpose = request.headers["HTTP_SPEC_PURPOSE"]
```

Comment:
SPEC_PURPOSE -> SEC_PURPOSE, maybe?

Reply:
:facepalm: I kept messing this up, and I copied the wrong one... Thanks, fixed!